src/TestCases/UnitTestCase.php

```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types=1);
+
+namespace Eonx\TestUtils\TestCases;
+
+use Eonx\TestUtils\TestCases\Traits\AssertTrait;
+
```

Comment:
Note for anyone else reading, this is what PHPUnit recommends. The manual registration method applies to test lifecycle hooks only, not assertions.

see https://phpunit.readthedocs.io/en/8.5/extending-phpunit.html#write-custom-assertions